Comanche055/TVCEXECUTIVE.agc

```diff
@@ -59,20 +60,22 @@
 # SUBROUTINES CALLED....NEEDLER, S40.15, MASSPROP, TASKOVER, IBNKCALL
 #
 # OTHER INTERFACES....
+#
 #      *TVCRESTART PACKAGE FOR RESTARTS
 #      *PITCHDAP, YAWDAP FOR VARIABLE GAINS AND ENGINE TRIM ANGLES
-#
 # ERASABLE INITIALIZATION REQUIRED....
+#
 #      *SEE TVCDAPON....TVCINIT4
-#      *VARK AND 1/CONACC (S40.15 OF R03)
```

Comment:
R03 was way off. Thats a bit weird. Theres no R03 text in that scan. I wonder what happened.

Reply:
I saw another that was way off. The transcribers must have had a bunch of papers laid out in front of them, and sometimes glanced at the wrong paper.